internal/servers/repository.go

```diff
@@ -67,18 +75,27 @@ func (r *Repository) listServersWithReader(ctx context.Context, reader db.Reader
 	opts := getOpts(opt...)
 	liveness := opts.withLiveness
 	if liveness == 0 {
-		liveness = defaultLiveness
+		liveness = DefaultLiveness
 	}
+
+	var where string
+	if liveness > 0 {
```

Comment:
I am not clear how `liveness` will ever `<= 0`.  Doesn't the `if` statement above prevent this?  Unless of course we change the `DefaultLiveness` to `0` at some point but that should never happen anyway...

Reply:
Reverted this as we actually use a negative number here to disable the default liveness setting.

Reply:
See https://github.com/hashicorp/boundary/blob/ebc0b96b0b7dc989202f8fba255de79e251f2164/internal/servers/options.go#L40-L47

Reply:
ty